src/main/java/homework_6/map_problems_generator/MapProblemsCollisionGenerator.java

```diff
@@ -0,0 +1,31 @@
+package homework_6.map_problems_generator;
+
+import java.util.HashMap;
+
+public class MapProblemsCollisionGenerator {
+
+    public static void main(String[] args) {
```

Comment:
Good example of mutable key problem, but where's the object, that makes 100% collisions in hashMap?